src/components/DrumPad.spec.js

```diff
@@ -0,0 +1,18 @@
+import DrumPad from './DrumPad.js';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+
+describe('DrumMachine', () => {
+  it('renders a section with sixteen buttons', () => {
+    const padClick = jest.fn()
+    render(<DrumPad handleClick={padClick} />);
+
+    const drumPad = screen.getByRole('button', { name: 'Drum Pad' });
```

Comment:
You only get the first button in the document. 
You could try this: expect(screen.getallByRole('button')).length.toBe(16).

Reply:
In this component i only have 1 button.
But you are right, i can test it on the DrumMachinePage.

Reply:
In the DrumPad component I have only one button.
But you are right, I can test this on the DrumMachinePage